cassandra-mesos-model/src/main/proto/io/mesosphere/mesos/frameworks/cassandra/model.proto

```diff
@@ -81,4 +94,24 @@ message SlaveErrorDetails {
 
 message CassandraNodeHealthCheckDetails {
     required bool healthy = 1;
+    optional CassandraNodeInfo info = 2;
```

Comment:
Perhaps in the future the health check task can be extended to specify the features we expect to be enabled and the state we expect them to be in.

Though I think that's probably a bit much right now.